democracy/models/base.py

```diff
@@ -124,10 +124,6 @@ class Commentable(models.Model):
 
     def recache_n_comments(self):
         new_n_comments = self.comments.count()
-        # if commentable has sections, include them in the total comment count
```

Comment:
Are you sure this is the right change? I imagined Hearings should tally the total number of comments in their Sections, so you could for instance sort by the most commentedest hearing.